packages/usa-in-page-navigation/src/usa-in-page-navigation.stories.js

```diff
@@ -42,3 +44,19 @@ TestCustomHeaderSelector.argTypes = {
 };
 
 export const TestHiddenHeaders = TestHiddenHeaderTemplate.bind();
+
+export const TestMinimumHeaders = TestMinimumHeaderTemplate.bind();
+TestMinimumHeaders.argTypes = {
+  headerLevels: {
+    defaultValue: "1",
+    name: "Number of headers on page",
+    options: ["1", "2", "3", "4"],
+    control: { type: "select" },
+  },
+  minimumHeaderCount: {
+    defaultValue: "1",
+    name: "Minimum number of headers required to show navigation",
+    options: ["1", "2", "3", "4"],
+    control: { type: "select" },
+  },
+};
```

Comment:
**issue:** Thank you for creating these test controls! For some reason I'm unable to use them. 
- Changing the control causes the template to reload without the JS
- Refreshing the page resets the control to it’s defaults

I've seen this JS behavior in other stories but the others keep the set value after reloading. Can you take a look at how the other in-page navigation stories are set up and see if we can get the test story to display correctly?

Reply:
Fixed in [18eeec9](https://github.com/uswds/uswds/pull/6205/commits/18eeec9d17b4b8ee251a10d595b869716c319efe)